be/src/exprs/vectorized/time_functions.cpp

```diff
@@ -173,13 +175,13 @@ ColumnPtr TimeFunctions::convert_tz_general(FunctionContext* context, const Colu
 
         int64_t timestamp;
         // TODO find a better approach to replace datetime_value.unix_timestamp
-        if (!ts_value.unix_timestamp(&timestamp, std::string(from_format.data, from_format.size))) {
+        if (!ts_value.unix_timestamp(&timestamp, timezone_hsscan, std::string(from_format.data, from_format.size))) {
```

Comment:
There is no need to add a new unix_timestamp function, we can use find_cctz_time_zone and unix_timstamp(timestamp, zone)

Reply:
OK